rosdep/base.yaml

```diff
@@ -4626,6 +4626,341 @@ libpcl-all-dev:
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-octree1.8]
+    focal: [libpcl-octree1.10]
+    jammy: [libpcl-octree1.12]
+libpcl-outofcore:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-outofcore1.11]
+    buster: [libpcl-outofcore1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-outofcore1.8]
+    focal: [libpcl-outofcore1.10]
+    jammy: [libpcl-outofcore1.12]
+libpcl-people:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-people1.11]
+    buster: [libpcl-people1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-people1.8]
+    focal: [libpcl-people1.10]
+    jammy: [libpcl-people1.12]
+libpcl-recognition:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-recognition1.11]
+    buster: [libpcl-recognition1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-recognition1.8]
+    focal: [libpcl-recognition1.10]
+    jammy: [libpcl-recognition1.12]
+libpcl-registration:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-registration1.11]
+    buster: [libpcl-registration1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-registration1.8]
+    focal: [libpcl-registration1.10]
+    jammy: [libpcl-registration1.12]
+libpcl-sample-consensus:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-sample-consensus1.11]
+    buster: [libpcl-sample-consensus1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-sample-consensus1.8]
+    focal: [libpcl-sample-consensus1.10]
+    jammy: [libpcl-sample-consensus1.12]
+libpcl-search:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-search1.11]
+    buster: [libpcl-search1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
+    bionic: [libpcl-search1.8]
+    focal: [libpcl-search1.10]
+    jammy: [libpcl-search1.12]
+libpcl-segmentation:
+  alpine: [pcl-libs]
+  debian:
+    bullseye: [libpcl-segmentation1.11]
+    buster: [libpcl-segmentation1.9]
+  fedora: [pcl]
+  freebsd: [libpcl]
+  gentoo: [sci-libs/pcl]
+  macports: [libpcl]
+  nixos: [pcl]
+  openembedded: [pcl@meta-ros-common]
+  rhel: [pcl]
+  slackware: [pcl]
+  ubuntu:
```

Comment:
What about bionic for libpcl-segmentation, libpcl-stereo, libpcl-surface, libpcl-tracking, libpcl-visualization?